test/Lint.lean

```diff
@@ -116,3 +116,112 @@ info: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20,
+which would be better stated with explicit decidability statements.
+Instead, use `open Classical in` for definitions or instances, the `classical` tactic for proofs.
+For theorem statements, either add missing decidability assumptions or use `open Classical in`.
+note: this linter can be disabled with `set_option linter.openClassical false`
+-/
+#guard_msgs in
+open Classical hiding choose
+
+/--
+warning: please avoid 'open (scoped) Classical' statements: this can hide theorem statements
+which would be better stated with explicit decidability statements.
+Instead, use `open Classical in` for definitions or instances, the `classical` tactic for proofs.
+For theorem statements, either add missing decidability assumptions or use `open Classical in`.
+note: this linter can be disabled with `set_option linter.openClassical false`
+-/
+#guard_msgs in
+open Classical hiding choose axiomOfChoice
+
+/--
+warning: please avoid 'open (scoped) Classical' statements: this can hide theorem statements
+which would be better stated with explicit decidability statements.
+Instead, use `open Classical in` for definitions or instances, the `classical` tactic for proofs.
+For theorem statements, either add missing decidability assumptions or use `open Classical in`.
+note: this linter can be disabled with `set_option linter.openClassical false`
+-/
+#guard_msgs in
+open Classical renaming choose -> foo, byCases -> bar
+
+-- Only opening specific items.
+/--
+warning: please avoid 'open (scoped) Classical' statements: this can hide theorem statements
+which would be better stated with explicit decidability statements.
+Instead, use `open Classical in` for definitions or instances, the `classical` tactic for proofs.
+For theorem statements, either add missing decidability assumptions or use `open Classical in`.
+note: this linter can be disabled with `set_option linter.openClassical false`
+-/
+#guard_msgs in
+open Classical (choose)
+
+-- `open scoped Classical` is also linted
+/--
+warning: please avoid 'open (scoped) Classical' statements: this can hide theorem statements
+which would be better stated with explicit decidability statements.
+Instead, use `open Classical in` for definitions or instances, the `classical` tactic for proofs.
+For theorem statements, either add missing decidability assumptions or use `open Classical in`.
+note: this linter can be disabled with `set_option linter.openClassical false`
+-/
+#guard_msgs in
+open scoped Classical
+
+/--
+warning: please avoid 'open (scoped) Classical' statements: this can hide theorem statements
+which would be better stated with explicit decidability statements.
+Instead, use `open Classical in` for definitions or instances, the `classical` tactic for proofs.
+For theorem statements, either add missing decidability assumptions or use `open Classical in`.
+note: this linter can be disabled with `set_option linter.openClassical false`
+-/
+#guard_msgs in
+open scoped Classical
```

Comment:
This one is repeated.  Is this intended?  Did you want to test something different instead?  Is it just a copy-paste oversight?

Reply:
Good catch! I think I wanted to adapt the tests for `open` (which handle several cases), but for `open scoped` there is no variant for hiding or renaming items, and this got left over. I've removed this.